lib/plugins/attributeAlias.js

```diff
@@ -93,7 +93,7 @@ function updateAttribute(entity, typeInformation, callback) {
     let attsArray = utils.extractAttributesArrayFromNgsi2Entity(entity);
     attsArray = attsArray.map(applyAlias(mappings));
     entity = utils.createNgsi2Entity(entity.id, entity.type, attsArray, true);
-    ngsiUtils.castJsonNativeAttributes(entity);
```

Comment:
Should the old function completely removed? Or it is used yet at some place?

Reply:
old funcion, now called castJsonNativeAttributesType is already used by ngsi-ld

Reply:
Maybe we can remove it after this PR https://github.com/telefonicaid/iotagent-node-lib/pull/1322

Reply:
PR #1322 still uses the old function `castJsonNativeAttributes`
https://github.com/telefonicaid/iotagent-node-lib/blob/5996da6bb9f1024ee7d52e805850f1ddd9788acd/lib/services/ngsi/entities-NGSI-LD.js#L1020

Reply:
@jason-fox should this aligned also in the NGSI-LD part of the code? I mean, to remove also the autocast functionality there, in the same way it has been removed in NGSIv2 in this PR.

If it is kept, maybe it would be better to put it in the NGSI-LD part instead that in the common `NGSIUtils`.

Reply:
@fgalan - yes, since #1322 has been merged, the LD here is as far as possible a clone of the v2 function, so changing both together makes sense.